### PR TITLE
provides essential info on the project

### PR DESCRIPTION
**Pull request by iychche**

### CODE REVIEW

**Comment by OlegDokuka** [comment]
@iychche I set up the DCO check, please make sure your commit is signed.

Best,
Oleh

**Comment by OlegDokuka** [comment]
Also, it would be great to have an initial discussion through something like `Road Map` issue including all the steps needed for the implementation (e.g. https://github.com/rsocket/rsocket-kotlin/issues/115). 
Also, all following PRs or discussions may be done in the separate Github issue but references in the `Road Map` one for clarity.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,24 @@
...
+## Strategy
+In phase one, the plan is to focus on the client APIs along with support for the following features
+
+### Transport
+[ ] WebSocket
+
+### Interaction Model
+[ ] Request/Response
+
+In the following phases, the plan is to implmement the server APIs along with support for
```
Comment (gabeb-intu) [inline README.md:14]: implement

**Comment by gabeb-intu** [comment]
> @iychche I set up the DCO check, please make sure your commit is signed. (it is organization policy to sign that you have rights to contribute)
> 
> Also, it worth mentioning that the implementation is planned for the client-side only (or both?)
> 
> Best,
> Oleh

We are only planning to implement for client side for now

**Comment by gabeb-intu** [comment]
> Also, it would be great to have an initial discussion through something like `Road Map` issue including all the steps needed for the implementation (e.g. [rsocket/rsocket-kotlin#115](https://github.com/rsocket/rsocket-kotlin/issues/115)). Following PRs or discussions may be done in the separate Github issue but being referenced in the `Road Map` one for clarity.
> 
> Following such a format will help to understand what is implemented as well as discuss all the details without the need to update readme every time in case something has changed. Alternatively, the GitHub wiki can be used https://github.com/rsocket/rsocket-swift/wiki

I like the wiki approach but we will discuss with the team and get back to you

**Comment by OlegDokuka** [comment]
@gabeb-intu do not treat it as something you mandated to do. It is just my advice from my experience

**Review by gabeb-intu — APPROVED** [review]

**Pull request merged**